src/FTPWindow.cpp

```diff
@@ -160,8 +160,9 @@ TFTPWindow::TFTPWindow(BRect frame, const char *name)
 	fRemoteFileView->AddStatusView(fItemCountView);
 
 	// CheckBox (Use This Connection)
-	const char* str = B_TRANSLATE("Use this connection");
-	fUseThisConnection = new BCheckBox("UseThisConnection", str, NULL, 0);
+	fUseThisConnection = new BCheckBox("UseThisConnection",
+									   B_TRANSLATE("Use this connection"),
```

Comment:
In my editor it looks good, but when I view the changes on GitHub the indentation looks awkward.